src/libraries/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -407,7 +408,7 @@ public void Name_Set_FullName_Invalid()
         {
```

Comment:
This test is testing the exact same as the last theory of `Name_Set_FullName` . It should be deleted instead.